front/pages/api/w/[wId]/spaces/[spaceId]/data_source_views/index.ts

```diff
@@ -172,14 +172,14 @@ async function handler(
     }
 
     case "POST": {
-      if (!space.canWrite(auth)) {
-        // Only admins, or builders who have to the space, can create a new view
+      if (!space.canWrite(auth) && !space.canAdministrate(auth)) {
```

Comment:
Shoud it be "canAdministrate" only ?

I'm a bit confused by write & administrate here, the verb should be related to the action to perform, not contextual imho.

Maybe canAdministrate should be canWrite + something else ?

Reply:
looking a bit more

Seems like canAdministrate = can manage (managed) datasource
canWrite = can manage (static) datasource

?

Should it be renamed to avoid confusion ?

Reply:
Or there is only canWrite for managing datasource and canAdministrate is related to something else.

Reply:
> Which is okay since, they can't read anything

Which is also quite weak :) They can just add themselves :)